code/modules/mob/mob.dm

```diff
@@ -346,6 +346,8 @@ mob/visible_message(message, self_message, blind_message, vision_distance = DEFA
 	else
 		result = A.examine(src) // if a tree is examined but no client is there to see it, did the tree ever really exist?
 
+	if(!result)
+		CRASH("Examine failed for [A] ([A.type]) in [AREACOORD(A)]")
```

Comment:
```suggestion
		CRASH("Examine failed for [A] ([A?.type]) in [AREACOORD(A)]")
```
You really don't want to runtime on a `CRASH` report.

Reply:
It would already have runtimed prior; either it does A.examine() or A.examine_more(). Still a good idea though, plus maybe an `if(!A) return` up top. I don't expect anything to delete mid-examine, *but* doing both can't hurt.